.buildkite/x-pack/pipeline.xpack.winlogbeat.yml

```diff
@@ -10,39 +12,136 @@ env:
   IMAGE_WIN_2022: "family/platform-ingest-beats-windows-2022"
   GCP_HI_PERF_MACHINE_TYPE: "c2d-highcpu-16"
   GCP_WIN_MACHINE_TYPE: "n2-standard-8"
-  BEATS_PROJECT_NAME: "x-pack/winlogbeat"
-
-  #Packaging
-  PACKAGING_PLATFORMS: "+all linux/amd64 linux/arm64 windows/amd64 darwin/amd64 darwin/arm64"
-  PACKAGING_ARM_PLATFORMS: "linux/arm64"
 
-  #Deps
-  ASDF_MAGE_VERSION: 1.15.0
+notify:
+  - github_commit_status:
+      context: "x-pack/winlogbeat"
 
 steps:
+  - group: "Mandatory Tests"
+    key: "xpack-winlogbeat-mandatory-tests"
+    steps:
+
+      - label: ":windows: Xpack/Winlogbeat Win-2019 Unit (MODULE) Tests"
+        key: "mandatory-win-2019-module-unit-tests"
+        command: |
+          Set-Location -Path x-pack/winlogbeat
+          mage build unitTest
+        env:
+          MODULE: $MODULE
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_WIN_2019}"
+          machine_type: "${GCP_WIN_MACHINE_TYPE}"
+          disk_size: 100
+          disk_type: "pd-ssd"
+        artifact_paths:
+          - "x-pack/winlogbeat/build/*.xml"
+          - "x-pack/winlogbeat/build/*.json"
+
+      - label: ":windows: Xpack/Winlogbeat Win-2016 Unit Tests"
+        command: |
+          Set-Location -Path x-pack/winlogbeat
+          mage build unitTest
+        key: "mandatory-win-2016-unit-tests"
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_WIN_2016}"
+          machine_type: "${GCP_WIN_MACHINE_TYPE}"
+          disk_size: 100
+          disk_type: "pd-ssd"
+        artifact_paths:
+          - "x-pack/winlogbeat/build/*.xml"
+          - "x-pack/winlogbeat/build/*.json"
 
-  - input: "Input Parameters"
-    key: "force-run-stages"
-    fields:
-    - select: "Winlogbeat - run_xpack_winlogbeat"
-      key: "run_xpack_winlogbeat"
-      options:
-        - label: "True"
-          value: "true"
-        - label: "False"
-          value: "false"
-      default: "false"
-    if: "build.source == 'ui'"
+      - label: ":windows: Xpack/Winlogbeat Win-2022 Unit Tests"
+        command: |
+          Set-Location -Path x-pack/winlogbeat
+          mage build unitTest
+        key: "mandatory-win-2022-unit-tests"
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_WIN_2022}"
+          machine_type: "${GCP_WIN_MACHINE_TYPE}"
+          disk_size: 100
+          disk_type: "pd-ssd"
+        artifact_paths:
+          - "x-pack/winlogbeat/build/*.xml"
+          - "x-pack/winlogbeat/build/*.json"
+
+  - group: "Extended Windows Tests"
+    key: "extended-win-tests"
+    if: build.env("BUILDKITE_PULL_REQUEST") == "false" || build.env("GITHUB_PR_LABELS") =~ /.*windows.*/
```

Comment:
```suggestion
    if: build.env("BUILDKITE_PULL_REQUEST") == "false" || build.env("GITHUB_PR_LABELS") =~ /.*[Ww]indows.*/
```

Reply:
by switching to `[Ww]indows` the extended tests will be triggered with both `:Windows` as well as the other `windows-XX` labels.